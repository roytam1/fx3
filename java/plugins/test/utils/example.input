#
# example.input
#
# This file shows, how you can use the input key values
#
key1= somebody@mail.1
key2= noone@mail.2
key3= anonymous@mail.3 (# This does not seems to be a comment)
