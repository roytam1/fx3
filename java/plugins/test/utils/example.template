#
# example.template
#
# This file shows, how you can use the 
# template values.
#

Welcome to the template. You can send a dummy mail to the
one man : <key1>. If you does like this example you can
send the email to <key2>, <key3> . Note, that all these
email addresses are in space - there's no such a domains.

Thank you.