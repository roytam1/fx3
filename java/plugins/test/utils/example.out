#
# example.template
#
# This file shows, how you can use the 
# template values.
#

Welcome to the template. You can send a dummy mail to the
one man :  somebody@mail.1. If you does like this example you can
send the email to  noone@mail.2,  anonymous@mail.3 (# This does not seems to be a comment) . Note, that all these
email addresses are in space - there's no such a domains.

